examples/falco/agent.py

```diff
@@ -102,6 +105,8 @@ def output(self, answer: str, success: bool) -> AgentOutput:
     def get_messages(self, task: str) -> list[AllMessageValues]:
         self.conv.reset()
         system_msg, task_msg = self.prompt.system(), self.prompt.task(task)
+        if self.vault is not None:
+            system_msg += "\n" + self.vault.instructions()
```

Comment:
cc @leo-notte I think we could think of a tool register component so that this register can automatically append the custom instructions for each of the register tools instead of doing it manually like here. But for later, just something to think about